include/rocksdb/env.h

```diff
@@ -473,16 +470,6 @@ class RandomAccessFile {
   virtual Status Read(uint64_t offset, size_t n, Slice* result,
                       char* scratch) const = 0;
 
-  // Used by the file_reader_writer to decide if the ReadAhead wrapper
-  // should simply forward the call and do not enact buffering or locking.
-  virtual bool ShouldForwardRawRequest() const {
```

Comment:
Just a note that these two public APIs seem to be referenced by at least one project internally at FB. cc: @siying